tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SQS/SendMessageAsyncIntegration.cs

```diff
@@ -52,16 +52,24 @@ public class SendMessageAsyncIntegration
             // for the InjectHeadersIntoMessage<TSendMessageRequest> call below
             var requestProxy = request.DuckCast<ISendMessageRequest>();
 
+            var queueName = AwsSqsCommon.GetQueueName(requestProxy.QueueUrl);
             var scope = AwsSqsCommon.CreateScope(Tracer.Instance, Operation, out var tags, spanKind: SpanKinds.Producer);
             if (tags is not null && requestProxy.QueueUrl is not null)
             {
                 tags.QueueUrl = requestProxy.QueueUrl;
-                tags.QueueName = AwsSqsCommon.GetQueueName(requestProxy.QueueUrl);
+                tags.QueueName = queueName;
             }
 
             if (scope?.Span.Context != null)
             {
-                ContextPropagation.InjectHeadersIntoMessage<TSendMessageRequest>(requestProxy, scope.Span.Context);
+                var dataStreamsManager = Tracer.Instance.TracerManager.DataStreamsManager;
+                if (dataStreamsManager != null && dataStreamsManager.IsEnabled)
+                {
+                    var edgeTags = new[] { "direction:out", $"topic:{queueName}", "type:sqs" };
+                    scope.Span.SetDataStreamsCheckpoint(dataStreamsManager, CheckpointKind.Produce, edgeTags, 0, 0);
```

Comment:
nit, can we add names to the `0` arguments, so it's clear what they are 🙂

Reply:
ah yes, I didn't do it because Rider does it for you, but it'd still help with github reviews.
<img width="315" alt="image" src="https://github.com/DataDog/dd-trace-dotnet/assets/1932410/da2fc176-9949-4a61-b258-fd2c9ffb9598">